copy-types.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
```

Comment:
I don't know if I understand - why do we need this file? `index.d.ts` lands in distribution package automatically just because it's added to `files` in `package.json` - wouldn't that happen for `entry.nostyle.d.ts` too?